tst/ordering.tst

```diff
@@ -1,34 +1,7 @@
 gap> START_TEST("ordering.tst");;
 
```

Comment:
You removed a comment sign, and now tests fail - please restore it or remove an empty line at all, otherwise tests fail.

Reply:
Thanks. Will be fixed.